hbase/versions.py

```diff
@@ -36,7 +36,7 @@
         "java-base": "11",
         "java-devel": "11",
         "async_profiler": "2.9",
-        "phoenix": "5.3.0-4afe457",
+        "phoenix": "5.3.0-ca21a87dd6",
```

Comment:
Can we add a comment next to this to show where it comes from?

Also, common short-hash is 7 characters long (unless there is a new recommendation to avoid collisions)?

```suggestion
        "phoenix": "5.3.0-ca21a87", # This is currently manually built and uploaded to Nexus
```